File: iniciante/python/1180.py
# -*- coding: utf-8 -*-

N = int(input())
entrada = input()
X = entrada.split()
menor_elem = int(X[0])
pos_menor = 0
for pos in range(1, len(X), 1):
    if menor_elem > int(X[pos]):
        menor_elem = int(X[pos])
        pos_menor = pos
print("Menor valor: %d" %menor_elem)
print("Posicao: %d" %pos_menor)

File: iniciante/c/1180.c
#include <stdio.h>

int main(){

	int N, menor_valor, i, posicao;
	scanf("%d", &N);
	int  X[N];
	for (i=0; i < N; i++){
	    scanf("%d", &X[i]);
	}
	menor_valor = X[0];
	for (i=1; i <N; i++){
		if (menor_valor > X[i]){
			menor_valor = X[i];
			posicao = i;
		}
	}
	printf("Menor valor: %d\n",menor_valor);
	printf("Posicao: %d\n",posicao);

    return 0;
}